clickable/clickable.yaml

```diff
@@ -0,0 +1,35 @@
+clickable_minimum_required: '8.1.0'
+framework: ubuntu-sdk-20.04
+builder: cmake
+
+build_args:
+- -D NCINE_STRIP_BINARIES=ON
+- -D NCINE_PREFERRED_BACKEND=GLFW
+- -D NCINE_LINUX_PACKAGE=jazz2.dubix
+- -D NCINE_PACKAGED_CONTENT_PATH=ON
+- -D NCINE_WITH_GLEW=OFF
+
+dependencies_target:
+- libopenal-dev
+- libdw-dev
+- libdwarf-dev
+- libglfw3-dev
+- libvorbisfile3
+- libopenmpt-dev
+- libgles-dev
+
+install_lib:
```

Comment:
If you want to go the SDL route, you can add `/opt/sdl/*so*` here to ship a patched SDL version that works with MIR.